File: swagger/models/commission_detail.py
from .base_model import BaseModel


class CommissionDetail(BaseModel):
	"""
	no description available

	Attributes:

		commission_type
		The type of commission earned. Possible values are:  *
		ItemStandardPercentageCommission  * ItemStandardFlatCommission  *
		ItemPromotionalPercentageCommission  * ItemPromotionalFlatCommission
		* StaffStandardPercentageCommission  * StaffStandardFlatCommission  *
		StaffPromotionalPercentageCommission  * StaffPromotionalFlatCommission

		commission_earnings
		The portion of `Earnings` earned by this `CommissionType`.
	"""

	input_map = {
		'CommissionType': 'commission_type',
		'CommissionEarnings': 'commission_earnings',
		}

	output_map = {
		'commission_type': 'CommissionType',
		'commission_earnings': 'CommissionEarnings',
		}

File: swagger/Models/CommissionDetail.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class CommissionDetail extends BaseModel
{
	/**
	 * The type of commission earned. Possible values are:  *
	 * ItemStandardPercentageCommission  * ItemStandardFlatCommission  *
	 * ItemPromotionalPercentageCommission  * ItemPromotionalFlatCommission
	 * * StaffStandardPercentageCommission  * StaffStandardFlatCommission  *
	 * StaffPromotionalPercentageCommission  * StaffPromotionalFlatCommission
	 * 
	 * @var string
	 */
	public $commissionType;
	/**
	 * The portion of `Earnings` earned by this `CommissionType`.
	 * 
	 * @var number
	 */
	public $commissionEarnings;

	protected function getInputMap()
	{
		return [
			'CommissionType' => 'commissionType',
			'CommissionEarnings' => 'commissionEarnings',
			];
	}

	protected function getOutputMap()
	{
		return [
			'commissionType' => 'CommissionType',
			'commissionEarnings' => 'CommissionEarnings',
			];
	}
}
